app/uk/gov/hmrc/platforminitiatives/connectors/ServiceDependenciesConnector.scala

```diff
@@ -48,10 +48,11 @@ class ServiceDependenciesConnector @Inject() (
   def getServiceDependency(
     group       : String,
     artefact    : String,
+    environment : String,
```

Comment:
"latest" is not technically an environment (hence the api naming of `flag`). Since the default behaviour when not provided is "latest", `getServiceDependency` could take environment as optional.
Also you could use a type for `Environment` rather than `String` for client safety.